src/gc/unix/gcenv.unix.cpp

```diff
@@ -77,14 +77,7 @@ static size_t g_RestrictedPhysicalMemoryLimit = 0;
 //  true if it has succeeded, false if it has failed
 bool GCToOSInterface::Initialize()
 {
-    // Calculate and cache the number of processors on this machine
-    int cpuCount = sysconf(_SC_NPROCESSORS_ONLN);
-    if (cpuCount == -1)
-    {
-        return false;
-    }
-
-    g_logicalCpuCount = cpuCount;
+    g_logicalCpuCount = PAL_GetLogicalCpuCountFromOS();
```

Comment:
Aha. I assumed gcenv.unix.cpp would be the implementation used by coreclr on Linux, but in fact that is gc.os.cpp...
So, this is the implementation used by corert.